src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,295 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+
+public class Duke {
+    static final int MAX_TASKS = 100;
+    static final String EXCEPTION_1 = "Empty Description";
+    static final String EXCEPTION_2 = "Other Command";
+    static final String EXCEPTION_3 = "Mark/Unmark Out of Bounds";
+    static final String EXCEPTION_4 = "File Not Found";
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int numberOfTasks = 0;
+
+    public static void printDivider() {
+        System.out.println("\t----------------------------------------------------");
+    }
+
+    public static void printGreeting() {
+        printDivider();
+        System.out.println("\tHello! I'm Albot, your new best friend! ᕙ(`▿´)ᕗ");
+        System.out.println("\tWhat can I do for you today, friend? ʕ•́ᴥ•̀ʔっ♡");
+        System.out.println();
+        printDivider();
+    }
+
+    public static void printExitMessage() {
+        printDivider();
+        System.out.println("\tAww, you're leaving already? I'm gonna miss you (╥﹏╥)");
+        System.out.println("\tBut anyways bye! Hope to see you again soon, friend! (ɔ◔‿◔)ɔ ♥");
+        System.out.println();
+        printDivider();
+    }
+
+    public static void printExceptionMessage(String exceptionType) {
+        printDivider();
+
+        switch (exceptionType) {
+        case EXCEPTION_1 :
+            //Empty description of a task
+            System.out.println("\t☹ OOPS!!! The description of a task cannot be empty...");
+            break;
+        case EXCEPTION_2 :
+            //Commands other than the ones recognized
+            System.out.println("\t☹ OOPS!!! I'm sorry, but I don't know what that means...");
+            break;
+        case EXCEPTION_3 :
+            //Mark/Unmark/Delete index out of bounds
+            System.out.println("\t☹ OOPS!!! Please provide a valid task index...");
+        case EXCEPTION_4:
+            //File not found
+            System.out.println("\t☹ OOPS!!! Unfortunately file is not found...");
+        }
+
+        System.out.println();
+        printDivider();
+    }
+
+    public static void readFile() throws FileNotFoundException {
+        File file = new File("duke.txt");
+        try {
+            boolean isExisting = file.createNewFile();
```

Comment:
Avoid unused variables like isExisting as it can trip up the reader and hinder readability. This boolean can be used to inform the user if the contents are being restored or if a new file was created, for example.

Code Quality Guideline: Do Not Trip Up Reader